### PR TITLE
(DOCSP-32289): Swift: Add offline realm access example to landing page

### PR DESCRIPTION
**Pull request by dacharyc**
### Jira

- https://jira.mongodb.org/browse/DOCSP-32289

### Staged Changes

- [Swift SDK landing page](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-32289/sdk/swift/)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/swift.txt [hunk 1]
```diff
@@ -277,3 +277,12 @@ Example Projects
       :icon-alt: Cloud Mobile Transfer Icon
 
       Learn best practices around handling Sync errors and client resets.
+
+   .. card::
+      :headline: Offline Realm Access
```
Comment (lindseymoore) [inline source/sdk/swift.txt:282]:
```suggestion
      :headline: Offline Login and Realm Access
```

Comment (lindseymoore) [reply source/sdk/swift.txt:282]: Changed title in table based on PM!

File: source/sdk/swift.txt [hunk 2]
```diff
@@ -277,3 +277,12 @@ Example Projects
...
+      :cta: Example app
+      :url: https://github.com/realm/realm-swift-samples/blob/main/OfflineLoginRealmAccess/README.md
+      :icon: /images/icons/branding_2023/Technical_REALM_Offline10x.png
+      :icon-alt: Realm Offline Icon
+
+      Open a synced realm offline.
```
Comment (lindseymoore) [inline source/sdk/swift.txt:288]:
```suggestion
      Login a Device Sync user and open a synced realm offline.
```
Also to make consistent with table.

**Review by lindseymoore — APPROVED** [review]
Small changes, but otherwise LGTM!

File: source/sdk/swift.txt (hunk 2 above)
Comment (dacharyc) [reply source/sdk/swift.txt:288]: I'm applying this feedback locally with a small tweak - in the description here, "Log in" should be the verb form. If we're using it this way in the table, we should change the description there to be grammatically correct.

Comment (dacharyc) [reply source/sdk/swift.txt]: I'm applying this feedback locally with a small tweak - in the description here, "Log in" should be the verb form. If we're using it this way in the table, we should change the description there to be grammatically correct.

**Pull request merged**